src/Data/Vec/Membership/Setoid.agda

```diff
@@ -52,8 +52,7 @@ module _ {p} {P : Pred A p} where
 
   find : ∀ {n} {xs : Vec A n} → Any P xs → ∃ λ x → x ∈ xs × P x
   find (here px)   = (_ , here refl , px)
-  find (there pxs) with find pxs
-  ... | (x , x∈xs , px) = (x , there x∈xs , px)
+  find (there pxs) = let (x , x∈xs , px) = find pxs in (x , there x∈xs , px)
```

Comment:
Redundant parentheses, on LHS *and* RHS!

Reply:
ok, this one I'll change.